pipeline-templates/job/deploy.yml

```diff
@@ -77,6 +78,18 @@ jobs:
               QnaApiAuthenticationClientSecret: $(QnaApiAuthenticationClientSecret)
               QnaApiAuthenticationClientSecret_Api: $(QnaApiAuthenticationClientSecret_Api)
               QnaApiAuthenticationClientSecret_Web: $(QnaApiAuthenticationClientSecret_Web)
+        - template: azure-pipelines-templates/deploy/step/app-role-assignments.yml@das-platform-building-blocks
+          parameters:
+            ServiceConnection: ${{ parameters.AppRoleAssignmentsServiceConnection }}
+            ResourceName: $(UiAppServiceName)
+            Tenant: $(Tenant)
+            IsMultiRepoCheckout: true
```

Comment:
Again needs doing for all environments

Reply:
@ChrisJWoodcock Are you sure this needs updated also? The new tasks are in `DeployResources` stage with which seems to run for all environments (as i can't see any conditions restricting environments to run)